ComfyUI/Dockerfile.local-cpu

```
@@ -0,0 +1,86 @@
+# Clones the ComfyUI Manager repository and checks out the latest release
+# remove gitignore to allow adding submodule into custom_nodes
+RUN rm .gitignore
+RUN git submodule add $COMFYUI_MANAGER_REPO custom_nodes/ComfyUI-Manager \
+    && cd custom_nodes/ComfyUI-Manager \
+    && git checkout tags/$COMFYUI_MANAGER_VERSION
+RUN --mount=type=cache,target=/root/.cache/uv \
+    uv pip install -r custom_nodes/ComfyUI-Manager/requirements.txt
+
+# Creating ComfyUI Manager config file
+RUN mkdir -p user/default/ComfyUI-Manager
+RUN <<EOF cat >> user/default/ComfyUI-Manager/config.ini
```

Hadolint (2.12.0)

[error] Missing space before ]. Fix any mentioned problems and try again.

(SC1072)